distributed/versions.py

```diff
@@ -98,34 +98,38 @@ def get_package_info(pkgs):
 
 
 def error_message(scheduler, workers, client, client_name="client"):
-    # we care about the required & optional packages matching
-    try:
-        client_versions = client["packages"]
-        versions = [("scheduler", scheduler["packages"])]
-        versions.extend((w, d["packages"]) for w, d in sorted(workers.items()))
-    except KeyError:
-        return (
-            "Version mismatch for dask.distributed. "
-            "The scheduler has version >= 1.28.0 "
-            "but some other component is less than this"
-        )
-
-    mismatched = defaultdict(list)
-    for name, vers in versions:
-        for pkg, cv in client_versions.items():
-            v = vers.get(pkg, "MISSING")
-            if cv != v:
-                mismatched[pkg].append((name, v))
-
-    if mismatched:
-        from .utils import asciitable
-
-        errs = []
-        for pkg, versions in sorted(mismatched.items()):
-            rows = [(client_name, client_versions[pkg])]
-            rows.extend(versions)
-            errs.append("%s\n%s" % (pkg, asciitable(["", "version"], rows)))
+    from .utils import asciitable
 
+    MISSING, UNKNOWN = "MISSING", "UNKNOWN"
+    scheduler_name = "scheduler"
+
+    nodes = {**{client_name: client}, **{scheduler_name: scheduler}, **workers}
```

Comment:
This seems needlessly complex.  Maybe the following instead?

```suggestion
    nodes = {client_name: client, "scheduler": scheduler, **workers}
```

Reply:
:+1: was just for symmetry...

Reply:
:heavy_check_mark: inlined